projects/igniteui-angular/src/lib/directives/for-of/for_of.directive.ts

```diff
@@ -850,27 +849,98 @@ export class IgxForOfDirective<T> implements OnInit, OnChanges, DoCheck, OnDestr
             this.sizesCache,
             0
         );
+
         if (newStart + this.state.chunkSize > count) {
             newStart = count - this.state.chunkSize;
         }
+
+        const prevStart = this.state.startIndex;
         const diff = newStart - this.state.startIndex;
         this.state.startIndex = newStart;
+
         if (diff) {
             this.onChunkPreload.emit(this.state);
             if (!this.isRemote) {
-                this.fixedApplyScroll();
+                const container = this.dc.instance._viewContainer.element.nativeElement as HTMLElement;
+                const activeElement = document.activeElement as HTMLElement;
+
+                // Remove focus in case the the active element is inside the view container.
+                // Otherwise we hit an exception while doing the 'small' scrolls swapping.
+                // For more information:
+                //
+                // https://developer.mozilla.org/en-US/docs/Web/API/Node/removeChild
+                // https://bugs.chromium.org/p/chromium/issues/detail?id=432392
+                if (container.contains(document.activeElement)) {
```

Comment:
Not sure how performant this check will be since the display container can be quite big (for vertical igxGridForOf it will contain all visible rows).
Perhaps it would be better to have this in moveApplyScrollNext/moveApplyScrollPrev and have it check  just the particular view that is about to be detached&inserted for focus. That way it will search only in that one view rather than in all views and it will also be executed only for small scrolls (as it is probably not needed for big scrolls).